### PR TITLE
fix(discord-utilities): fixed the export of TwemojiRegex

### PR DESCRIPTION
**Pull request by favna**
Be gone lib that never updates

### CODE REVIEW

File: .github/workflows/auto-updater.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          GITHUB_USER: github-actions[bot]
+          GITHUB_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git add .;
+          if git diff-index --quiet HEAD --; then
+            echo "No changes to commit, exiting with code 0"
+            exit 0;
+          else
+            git remote set-url origin "https://${GITHUB_TOKEN}:x-oauth-basic@github.com/${GITHUB_REPOSITORY}.git";
```
Comment (vladfrangu) [inline .github/workflows/auto-updater.yml:39]: does this work?

File: scripts/twemoji-regex-updater.mjs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+url.searchParams.append('path', 'src/lib/regex.js');
+url.searchParams.append('since', timestamp);
+
+const [commits, { default: ciData }] = await Promise.all([
+	fetch(url, FetchResultTypes.JSON), //
+	import(shaTrackerFileUrl, { assert: { type: 'json' } }) //
+]);
+
+const data = { sha: commits.length ? commits[0].sha : null, length: commits.length };
+if (!data) {
```
Comment (vladfrangu) [inline scripts/twemoji-regex-updater.mjs:45]: Never not true

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: .github/workflows/auto-updater.yml (hunk 1 above)
Comment (favna) [reply .github/workflows/auto-updater.yml:39]: Yes I use it with graphql-pokemon https://github.com/favware/graphql-pokemon/runs/6766851881?check_suite_focus=true

Comment (vladfrangu) [reply .github/workflows/auto-updater.yml:39]: Your hooks there aren't executable btw 😄

File: scripts/twemoji-regex-updater.mjs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+const writePromises = [
+	//
+	writeFile(twemojiRegexFileUrl, `${filePrefix}${regexFromWeb}${fileSuffix}`)
+];
+
+if (data.sha) writePromises.push(writeFile(shaTrackerFileUrl, JSON.stringify({ ...ciData, twemojiRegexLastSha: data.sha })));
+
+await Promise.all(writePromises);
+
+console.log(green(`Successfully wrote updated learnsets data to file; Latest SHA ${data.sha}`));
```
Comment (kyranet) [inline scripts/twemoji-regex-updater.mjs:71]: Nice, we got twemoji with machine learning

**Review by kyranet — CHANGES_REQUESTED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

File: scripts/twemoji-regex-updater.mjs (hunk 2 above)
Comment (favna) [reply scripts/twemoji-regex-updater.mjs]: weird must be an artifact from an earlier write up in gqlp pokemon

**Pull request merged**